aiolifx/fixtures/device_features.py

```diff
@@ -0,0 +1,22 @@
+from enum import Enum
+
+
+class DeviceFeatures(Enum):
+    INFO = "Info"
+    FIRMWARE = "Firmware"
+    WIFI = "Wifi"
+    UPTIME = "Uptime"
+    POWER = "Power"
+    WHITE = "White"
+    COLOR = "Color"
+    PULSE = "Pulse"
+    HEV_CYCLE = "HEV Cycle"
+    HEV_CONFIGURATION = "HEV Configuration"
+    MULTIZONE_FIRMWARE_EFFECT = "Get Multizone Firmware Effect"
+    MULTIZONE_FIRMWARE_EFFECT_START_STOP = "Start/Stop Firmware Effect"
+    MATRIX_FIRMWARE_EFFECT = "Get Matrix Firmware Effect"
+    MATRIX_FIRMWARE_EFFECT_START_STOP = "Start/Stop Firmware Effect"
+    RELAYS = "Relays"
+    BUTTONS = "Buttons"
+    BUTTON_CONFIG = "Button Config"
+    REBOOT = "Reboot"
```

Comment:
I think this list should match the list of capabilities in the `product_defs.py` file as that's what we have to work with programatically and it's what's published by LIFX in their product definition file.